roBa にて zmk-layout-shift 適用時に Build Error 発生

### Description
roBa にて zmk-layout-shift 適用時に Build Error 発生 

### Environment
以下の roBa Github Action での Build
https://github.com/tokinarazu/zmk-config-roBa/actions/runs/15659350412/job/44114840736#step:11:286

エラー内容は、以下
```
[142/371] Building C object zephyr/CMakeFiles/zephyr.dir/tmp/zmk-config/zmk-layout-shift/src/behavior_layout_shift_key_press.c.obj
FAILED: zephyr/CMakeFiles/zephyr.dir/tmp/zmk-config/zmk-layout-shift/src/behavior_layout_shift_key_press.c.obj 
...(省略)...
/tmp/zmk-config/zmk-layout-shift/src/behavior_layout_shift_key_press.c:103:5: warning: braces around scalar initializer
  103 |     .parameter_metadata = {
      |     ^
/tmp/zmk-config/zmk-layout-shift/src/behavior_layout_shift_key_press.c:103:5: note: (near initialization for 'behavior_layout_shift_key_press_driver_api.parameter_metadata')
/tmp/zmk-config/zmk-layout-shift/src/behavior_layout_shift_key_press.c:104:9: error: field name not in record or union initializer
  104 |         .count = 1,
      |         ^
compilation terminated due to -Wfatal-errors.
```

解消方法はありますでしょうか。

--- a/src/behavior_layout_shift_key_press.c
+++ b/src/behavior_layout_shift_key_press.c
@@ -95,17 +95,33 @@ static int on_layout_shift_key_press_binding_released(struct zmk_behavior_bindin
     );
 }
 
+#if IS_ENABLED(CONFIG_ZMK_BEHAVIOR_METADATA)
+static const struct behavior_parameter_value_metadata keycode_value_metadata[] = {
+    {
+        .display_name = "Keycode",
+        .type = BEHAVIOR_PARAMETER_VALUE_TYPE_HID_USAGE,
+    },
+};
+
+static const struct behavior_parameter_metadata_set keycode_parameter_set = {
+    .param1_values = keycode_value_metadata,
+    .param1_values_len = ARRAY_SIZE(keycode_value_metadata),
+};
+
+static const struct behavior_parameter_metadata_set metadata_sets[] = {keycode_parameter_set};
+
+static const struct behavior_parameter_metadata layout_shift_key_press_parameter_metadata = {
+    .sets_len = ARRAY_SIZE(metadata_sets),
+    .sets = metadata_sets,
+};
+#endif
+
 static const struct behavior_driver_api behavior_layout_shift_key_press_driver_api = {
     .binding_pressed = on_layout_shift_key_press_binding_pressed,
     .binding_released = on_layout_shift_key_press_binding_released,
     .locality = BEHAVIOR_LOCALITY_CENTRAL,
 #if IS_ENABLED(CONFIG_ZMK_BEHAVIOR_METADATA)
-    .parameter_metadata = {
-        .count = 1,
-        .types = {
-            ZMK_BEHAVIOR_PARAM_TYPE_KEYCODE,
-        },
-    },
+    .parameter_metadata = &layout_shift_key_press_parameter_metadata,
 #endif
 };
 

